File: android/app/src/main/java/com/neostudios/zest/domain/model/MealKit.kt
package com.neostudios.zest.domain.model

import com.google.firebase.firestore.DocumentId

data class MealKit(
    @DocumentId val id: String = "",
    val name: String = "",
    val description: String = "",
    val imageUrl: String = "",
    val price: Double = 0.0,
    val recipes: List<MealKitRecipe> = emptyList(),
    val availableDates: List<String> = emptyList() // YYYY-MM-DD
)

data class MealKitRecipe(
    val recipeId: String = "",
    val servings: Int = 0
)

File: web/src/types/mealKit.ts
export interface MealKit {
  id: string;
  name: string;
  description: string;
  imageUrl: string;
  price: number;
  recipes: MealKitRecipe[];
  availableDates: string[]; // YYYY-MM-DD
}

export interface MealKitRecipe {
  recipeId: string;
  servings: number;
}
